### PR TITLE
Update for spark 2.2.2 release

### PR DESCRIPTION
**Pull request by tgravescs**
Release process for Spark 2.2.2 say to update the test.

### CODE REVIEW

**Comment by tgravescs** [comment]
test this please

**Comment by tgravescs** [comment]
@vanzin @gatorsmile

**Review by felixcheung — APPROVED** [review]

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala [hunk 1]
```diff
@@ -160,7 +160,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
 
 object PROCESS_TABLES extends QueryTest with SQLTestUtils {
   // Tests the latest version of every release line.
-  val testingVersions = Seq("2.0.2", "2.1.2", "2.2.1")
+  val testingVersions = Seq("2.0.2", "2.1.2", "2.2.2")
```
Comment (dongjoon-hyun) [inline sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala]: @tgravescs . Could you replace 2.1.2 with 2.1.3, too?

cc @vanzin (2.1.3 release manager).

Comment (tgravescs) [reply sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala]: updated

**Review by dongjoon-hyun — APPROVED** [review]
Thank you. +1, LGTM.

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by HyukjinKwon** [comment]
oops, branch-2.2 doesn't have the fix yet `https://github.com/apache/spark/pull/21779`.

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by tgravescs** [comment]
merged to branch-2.2

**Pull request closed without merging**